stage/pages/lms/course_page_lms.py

```diff
@@ -0,0 +1,9 @@
+from edxapp_pages.lms.course_info import CourseInfoPage
```

Comment:
Yes it makes sense to add it in edx-platform, should we add another ticket for this?